titus-common/src/main/java/com/netflix/titus/common/model/sanitizer/internal/ConstraintValidatorContextWrapper.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.common.model.sanitizer.internal;
+
+import javax.validation.ConstraintValidatorContext;
+
+public class ConstraintValidatorContextWrapper {
```

Comment:
Replace this class with `Function< String, ConstraintViolationBuilder >`, and the default implementation as here?

Reply:
Thank you for the feedback. Please review the updates.